### PR TITLE
Issue #13345: Enable Class Type Parameter Name Check Example Test Class

### PR DESCRIPTION
**Pull request by piyush-1234**
Issue: #13345

### CODE REVIEW

File: src/xdocs/checks/naming/classtypeparametername.xml [hunk 1]
```diff
@@ -66,11 +72,17 @@ class MyClass5&lt;RequestT&gt; {} // violation
...
         <p id="Example2-code">Example:</p>
         <source>
-class MyClass1&lt;T&gt; {}        // violation
-class MyClass2&lt;t&gt; {}        // violation
-class MyClass3&lt;abc&gt; {}      // violation
-class MyClass4&lt;LISTENER&gt; {} // OK
-class MyClass5&lt;RequestT&gt; {} // violation
+class Example2 {
+  class MyClass1&lt;T&gt; {}        // violation
+
```
Comment (romani) [inline src/xdocs/checks/naming/classtypeparametername.xml:77]: Please remove empty lines, let's keep examples compact

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/checks/naming/classtypeparametername.xml (hunk 1 above)
Comment (piyush-1234) [reply src/xdocs/checks/naming/classtypeparametername.xml]: done

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/classtypeparametername/Example3.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.classtypeparametername;
+
+// xdoc section -- start
+class Example3 {
+  class MyClass1<T> {}
+
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/classtypeparametername/Example3.java:16]: Remove lines in example files

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/classtypeparametername/Example3.java (hunk 2 above)
Comment (piyush-1234) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/classtypeparametername/Example3.java]: done

**Review by romani — APPROVED** [review]
Ok to merge

File: src/xdocs/checks/naming/classtypeparametername.xml [hunk 3]
```diff
@@ -87,11 +91,13 @@ class MyClass5&lt;RequestT&gt; {} // violation
         </source>
         <p id="Example3-code">Example:</p>
         <source>
-class MyClass1&lt;T&gt; {}        // violation
-class MyClass2&lt;t&gt; {}        // violation
-class MyClass3&lt;abc&gt; {}      // violation
-class MyClass4&lt;LISTENER&gt; {} // violation
-class MyClass5&lt;RequestT&gt; {} // OK
+class Example3 {
+  class MyClass1&lt;T&gt; {}
```
Comment (rnveach) [inline src/xdocs/checks/naming/classtypeparametername.xml:95]: Why isn't this a violation anymore?

Comment (romani) [reply src/xdocs/checks/naming/classtypeparametername.xml:95]: It was mistake of original doc.
Previously it was just text with any content, now it actual behavior.

Regexp allow just `T`.

**Review by rnveach — APPROVED** [review]

**Pull request merged**